test.sh

```diff
@@ -6,6 +6,7 @@ function finish {
   docker-compose down -v
 }
 trap finish EXIT
+finish
```

Comment:
We don't need to call finish here, line 8 means "run function finish on EXIT signal".

Some more context - http://redsymbol.net/articles/bash-exit-traps/

Reply:
thanks. silly typo!